### ISSUE TITLE
The treeview plugin prevents deleting files in the space

### ISSUE THREAD

**Issue by tmichela** [joekrill/silverbullet-treeview#11]
This error gets thrown on the backend:

```bash
silverbullet-1  | Deleting file Library/Extra/Snippet/taskp.md
silverbullet-1  | An exception was thrown as a result of invoking function showIfEnabled error: Unregistered syscall clientStore.get
silverbullet-1  | Error dispatching event page:deleted to treeview.showIfEnabled: Unregistered syscall clientStore.get
silverbullet-1  | Error deleting attachment Error: Unregistered syscall clientStore.get
silverbullet-1  |     at wa.onMessage (file:///silverbullet.js:117:27964)
silverbullet-1  |     at Worker.worker.onmessage (file:///silverbullet.js:117:27483)
silverbullet-1  |     at Worker.wrappedHandler (ext:deno_web/02_event.js:1401:12)
silverbullet-1  |     at innerInvokeEventListeners (ext:deno_web/02_event.js:754:7)
silverbullet-1  |     at invokeEventListeners (ext:deno_web/02_event.js:801:5)
silverbullet-1  |     at dispatch (ext:deno_web/02_event.js:658:9)
silverbullet-1  |     at Worker.dispatchEvent (ext:deno_web/02_event.js:1043:12)
silverbullet-1  |     at Worker.#pollMessages (ext:runtime/11_workers.js:258:12)
silverbullet-1  |     at eventLoopTick (ext:core/01_core.js:169:7)
```

**Comment by joekrill** [comment]
Thanks for the report. I'll get a fix for this out today.

**Comment by joekrill** [comment]
Sorry for the delay on this. I believe this should be fixed by daf7a57e1bc7ea51f0ce00feab6dcd78aa39f7f8.
